.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: "CITests"
+
+on:
+  push
+
+jobs:
+  citests:
+    name: CI-Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          # We must fetch at least the immediate parents so that if this is
+          # a pull request then we can checkout the head.
+          fetch-depth: 2
+          submodules: 'recursive'
+
+      - run: |
+          sudo apt-get install -y libssl-dev
```

Comment:
Not familiar with GitHub Action, but at very least you should install gdb too:
sudo apt-get install -y libssl-dev gdb